### DISCUSSION TITLE
Autotiling

### DISCUSSION THREAD

**Discussion by piercoder** [rxhanson/Rectangle#1225] [Ideas]
Would you be open to including an autotiling feature in a future release?

**Comment by rxhanson** [comment]
I don't have personal plans to add autotiling to Rectangle, although it is on my backlog for Rectangle Pro. I'm not opposed to merging a pull request for it, as long as it doesn't affect any of the existing functionality and is only configured via terminal command at first (not added to the UI in the first iteration).
